influx_mysql/src/reflists.nim

```diff
@@ -52,7 +53,8 @@ when not defined(disablereflists) and (compileOption("gc", "refc") or compileOpt
             raise newException(Exception, "Cannot allocate memory!")
 
         GC_ref(value)
-        result[] = (next: nil, value: cast[pointer](value))
+        result.next = nil
```

Comment:
Since values allocated by `new` are guaranteed to be zero-initialized, can this line be removed as well?